var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -9,21 +9,21 @@ class Mercury(CMakePackage):
     """Mercury is a C library for implementing RPC, optimized for HPC"""
 
     homepage = 'https://mercury-hpc.github.io/'
-    url = 'https://github.com/mercury-hpc/mercury/archive/0.9.0.tar.gz'
+    url = 'https://github.com/mercury-hpc/mercury/releases/download/v1.0.1/mercury-1.0.1.tar.bz2'
     git = 'https://github.com/mercury-hpc/mercury.git'
 
     version('develop', branch='master', submodules=True)
```

Comment:
Now that #1983 has finally been merged, you can actually rename this version to `master` if you want and it will still sort correctly.

Reply:
Hmm ok I'm not sure what this implies, should I use a URL that points to the master archive tarball?

Reply:
Nope. You can keep it as a download from git. Previously, Spack sorted `master` as being older than all other releases, but with #1983, `master` and `develop` are both treated as the newest versions. So you can do:
```suggestion
    version('master', branch='master', submodules=True)
```
and it should just work.

Reply:
oh I see thanks